tests/bud.bats

```diff
@@ -4473,15 +4475,15 @@ EOM
   _prefetch alpine
   _prefetch debian
 
-  run_buildah build --build-arg base=alpine --build-arg toolchainname=busybox --build-arg destinationpath=/tmp --pull=false $WITH_POLICY_JSON -f $BUDFILES/from-with-arg/Containerfile .
+  run_buildah build --build-arg base=alpine --build-arg toolchainname=busybox --build-arg destinationpath=/tmp --pull=ifmissing $WITH_POLICY_JSON -f $BUDFILES/from-with-arg/Containerfile .
   expect_output --substring "FROM alpine"
   expect_output --substring 'STEP 4/4: COPY --from=\$\{toolchainname\} \/ \$\{destinationpath\}'
   run_buildah rm -a
 }
 
 @test "bud preserve rootfs for --mount=type=bind,from=" {
   _prefetch alpine
-  run_buildah build --build-arg NONCE="$(date)" --layers --pull=false $WITH_POLICY_JSON -f Dockerfile.3 $BUDFILES/cache-stages
+  run_buildah build --build-arg NONCE="$(date)" --layers --pull=ifmissing $WITH_POLICY_JSON -f Dockerfile.3 $BUDFILES/cache-stages
```

Comment:
This one shouldn't need changing - the only base image is prefetched.  Does it start to fail without this change?

Reply:
Got a little carried away.